user-manual/access-content/browse.rst

```diff
@@ -1097,8 +1098,8 @@ browse page. Each is described below.
    :term:`administrator` can also configure application-wide defaults for public
    and authenticated users via **Admin > Settings** - for more information, see:
    :ref:`sort-browser-users` and :ref:`sort-browser-anonymous`.
-3. The number of descriptions associated with a particular term as an
-   :term:`access point` is listed in the "Results" column.
+3. The number of descriptions and authority records associated with a particular 
+   term as an :term:`access point` are listed in the "Results" columns.
```

Comment:
"are listed in the *Archival description count* and *Authority record count* columns"

Reply:
There no longer is a "Results" column.

Reply:
Good point, I'll make this change.